src/Vector3.h

```diff
@@ -393,11 +393,9 @@ class VectorHasher
 	{
 		// Guaranteed to have no hash collisions for any 128x128x128 area
 		size_t Hash = 0;
-		Hash ^= static_cast<size_t>(a_Vector.x);
-		Hash <<= 8;
-		Hash ^= static_cast<size_t>(a_Vector.y);
-		Hash <<= 8;
-		Hash ^= static_cast<size_t>(a_Vector.z);
+		Hash ^= a_Vector.x + 0x9e3779b9 + (Hash << 6) + (Hash >> 2);
+		Hash ^= a_Vector.y + 0x9e3779b9 + (Hash << 6) + (Hash >> 2);
+		Hash ^= a_Vector.z + 0x9e3779b9 + (Hash << 6) + (Hash >> 2);
```

Comment:
> @tigerw how does playing around with the VECTOR's hash function help you at all?

That is, a very good question, come to think of it. I was using vectors to store chunk coordinates during testing.